pkg/apis/eventing/v1alpha1/trigger_validation_test.go

```diff
@@ -74,6 +74,10 @@ var (
 	invalidInjectionAnnotation = "disabled"
 	injectionAnnotationPath    = fmt.Sprintf("metadata.annotations[%s]", InjectionAnnotation)
 )
+const (
+	validLabelName   = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
```

Comment:
could we perhaps indicate in the name of the const that the max. of chars is reached ? 

same for the `invalidLabelName` ?

Reply:
Sure. Please check again if the name is ok. Open to discuss also other names